src/main/java/core/basesyntax/SquareSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class SquareSupplier {
+    private static final RandomGeometricValue randomSize = new RandomGeometricValue();
+    private static final ColorSupplier color = new ColorSupplier();

```

Comment:
The field 'color' should not be a constant. When you define a constant it should be named in uppercase. Also think about renaming it to `colorSupplier`